src/app/loadout-builder/process-worker/process-utils.ts

```diff
@@ -107,6 +107,8 @@ function getEnergyCounts(modsOrItems: (ProcessMod | null | ProcessItemSubset)[])
       case DestinyEnergyType.Void:
         voidCount += 1;
         break;
+      default:
+        break;
```

Comment:
yuck

Reply:
Agree, I don't think there is a way out of it though and the rule to ensure a default is handy.

Reply:
it's a very handy rule no argument, just ugly.

also i would like standardization between `default:break;` here, and `default:}` in objective.tsx

Reply:
I think this could just be `default:}` here too.